src/main/kotlin/lotto/domain/PrizeResult.kt

```diff
@@ -0,0 +1,26 @@
+package lotto.domain
+
+enum class PrizeResult(val matchCount: Int, val prize: Int) {
+
+    THREE_MATCH(3, 5_000),
+    FOUR_MATCH(4, 50_000),
+    FIVE_MATCH(5, 1_500_000),
+    BONUS_MATCH(5, 30_000_000),
+    SIX_MATCH(6, 2_000_000_000),
+    ZERO_MATCH(0, 0);
```

Comment:
로또의 순위는 1 등부터 5 등까지입니다. 아래의 글이 도움 되면 좋겠어요.
https://dhlottery.co.kr/gameResult.do?method=myWin

```suggestion
    FIFTH(3, 5_000),
    FORTH(4, 50_000),
    THIRD(5, 1_500_000),
    SECOND(5, 30_000_000),
    FIRST(6, 2_000_000_000),
    MISS(0, 0);
```